test/support/schemas/hyper_schema.json

```diff
@@ -0,0 +1,93 @@
+{
+  "$schema": "http://json-schema.org/draft-04/hyper-schema",
```

Comment:
@bf4 yes, this was created using the prmd gem. I will take a look at your comment.